cobalt/site/docs/reference/starboard/modules/14/media.md

```diff
@@ -691,15 +702,15 @@ bool SbMediaIsBufferUsingMemoryPool()
 Communicate to the platform how far past `current_playback_position` the app
 will write audio samples. The app will write all samples between
 `current_playback_position` and `current_playback_position` + `duration`, as
-soon as they are available. The app may sometimes write more samples than that,
-but the app only guarantees to write `duration` past `current_playback_position`
-in general. The platform is responsible for guaranteeing that when only
-`duration` audio samples are written at a time, no playback issues occur (such
-as transient or indefinite hanging). The platform may assume `duration` >= 0.5
-seconds.
+soon as they are available (during is in microseconds). The app may sometimes
+write more samples than that, but the app only guarantees to write `duration`
+past `current_playback_position` in general. The platform is responsible for
+guaranteeing that when only `duration` audio samples are written at a time, no
+playback issues occur (such as transient or indefinite hanging). The platform
+may assume `duration` >= 0.5 seconds.
 
 #### Declaration
 
 ```
-void SbMediaSetAudioWriteDuration(SbTime duration)
+void SbMediaSetAudioWriteDuration(int64_t duration)
```

Comment:
Same as above - SbTime changes should have been #ifdeffed for Starboard 16 only.